.travis.yml

```diff
@@ -47,7 +47,7 @@ before_install:
 install: true
 before_script: true
```

Comment:
in commit message "one translation until" -> "one translation unit"

Reply:
Fixed.